queryfilter/queryfilter.py

```diff
@@ -1,13 +1,57 @@
 from __future__ import absolute_import
 
 from .base import FieldFilter
+from .utils import is_same_class
 
 
 class QueryFilter(object):
+    filter_candidates = {}
+
+    @classmethod
+    def get_filter_key_from_type_and_condition(cls, filter_type,
+                                               filter_condition):
+        return "{}.{}".format(filter_type, filter_condition)
```

Comment:
Generally it will be used like `cls.get_filter_key_from_type_and_condition(filter_type, filter_condition)` so it can just be `cls.get_filter_key(filter_type, filter_condition)` almost same clear.